internal/version/releases.go

```diff
@@ -0,0 +1,73 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package version
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+)
+
+type Releases []Release
+
+type Release struct {
+	TagName string `json:"tag_name"`
+}
+
+var DefaultReleaseProvider = getReleaseData(&http.Client{Timeout: time.Second * 3}, "https://api.github.com/repos/dynatrace/dynatrace-configuration-as-code/releases")
+
+func GetLatestVersion(releasesProvider func() (Releases, error)) (Version, error) {
+	releases, err := releasesProvider()
+	if err != nil {
+		return UnknownVersion, err
+	}
+	if len(releases) <= 0 {
+		return UnknownVersion, fmt.Errorf("no monaco Releases found")
+	}
+
+	tagName := releases[0].TagName // Assuming Releases are sorted in descending order
```

Comment:
Pls, don't assume. 
We have two possibilities: Either always query the last one (could be a backport in the future, currently this is not the case), or check properly for the correct latest version